app/controllers/users.js

```diff
@@ -32,3 +33,13 @@ exports.signIn = (req, res, next) => {
     return next(error);
   }
 };
+
+exports.listUsers = async (req, res, next) => {
+  try {
+    const { rawListUsers, page, limit } = await userService.listUsers(req.query.page, req.query.limit);
+    const listUsers = serializeUsers(rawListUsers, page, limit);
```

Comment:
You can do: 
return res.status(200).send(serializeUsers(rawListUsers, page, limit));

and avoid to define a var listUsers